File: Day-33/Web Browser OBJ-C/Web Browser OBJ-C/ViewController.m
//
//  ViewController.m
//  Web Browser OBJ-C
//
//  Created by Bronson Dupaix on 3/16/16.
//  Copyright © 2016 Bronson Dupaix. All rights reserved.
//

@import WebKit;
#import "ViewController.h"

@interface ViewController ()

@property (nonatomic, strong) WKWebView* webView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.webView = [[WKWebView alloc] initWithFrame:self.view.bounds];
    
    self.webView.backgroundColor = [UIColor blueColor];
    
    NSLog(@"%@", NSStringFromCGRect(self.view.bounds));
    
    [self.view addSubview:self.webView];
    
    
    NSURL * url = [NSURL URLWithString:@"http://www.google.com"];
    
    [self.webView loadRequest: [[NSURLRequest alloc] initWithURL:url]];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Day-03/TextField- Day 03/TextField- Day 03/ViewController.swift
//
//  ViewController.swift
//  TextField- Day 03
//
//  Created by Bronson Dupaix on 2/3/16.
//  Copyright © 2016 Bronson Dupaix. All rights reserved.
//

import UIKit

class ViewController: UIViewController, UITextFieldDelegate {

    @IBOutlet weak var nameTextField: UITextField!
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        func textFieldShouldReturn(textField: UITextField) -> Bool {
            print("textFieldShouldReturn")
            
            return true
        }
      
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

